### ISSUE TITLE
module 'jwt' has no attribute 'PyJWTError'

### ISSUE THREAD

**Issue by aidar-aminev** [pennersr/django-allauth#3873]
module 'jwt' has no attribute 'PyJWTError'

**Comment by pennersr** [comment]
What happens when you open a python shell and run:

```
>>> import jwt
import jwt
>>> jwt.__version__
jwt.__version__
'2.8.0'
>>> jwt.PyJWTError
jwt.PyJWTError
<class 'jwt.exceptions.PyJWTError'>
```

**Comment by krystofbe** [comment]
I encountered a similar issue where I received the error `module 'jwt' has no attribute 'PyJWTError'`. After investigating, I found that the problem was due to having the `jwt` package installed instead of `pyjwt`. 

Django Allauth requires the `pyjwt` package. To resolve this, I removed the `jwt` package and installed `pyjwt`:

```bash
pip uninstall jwt
pip install pyjwt
```

After making this change, the issue was resolved, and Django Allauth worked as expected.
